### PR TITLE
Delete data associated with NaN lat/long coordinates

### PR DESCRIPTION
**Pull request by xhoenner**
Addresses error described in https://github.com/aodn/content/issues/522

### CODE REVIEW

File: AODN/AODN-WAVE-NRT/ARDC_API_NRT/ardc_nrt/ardc_sofar_nrt.py [hunk 1]
```diff
@@ -67,6 +67,7 @@ def process_wave_source_id(source_id, incoming_path=None):
             return
 
         if data is not None:
+            data = data[data['latitude'].notnull()]
```
Comment (bpasquer) [inline AODN/AODN-WAVE-NRT/ARDC_API_NRT/ardc_nrt/ardc_sofar_nrt.py:70]: there may be instance when longitude is null  so worth also excluding these values

Comment (xhoenner) [reply AODN/AODN-WAVE-NRT/ARDC_API_NRT/ardc_nrt/ardc_sofar_nrt.py]: Thanks @bpasquer. Hopefully addressed your comments via https://github.com/aodn/data-services/pull/1208/commits/06969cc4f401474d857736920e02f077d382d9a0

**Comment by lbesnard** [comment]
could you also fix the 2 github action files in this repo as done in 
https://github.com/aodn/aodn_cloud_optimised/commit/2193da40a0697c861183d422a0b6afe43c42cba8

so that the unittest can run ? 

there have been some changes lately, and i got the same bug

**Comment by xhoenner** [comment]
> could you also fix the 2 github action files in this repo as done in [aodn/aodn_cloud_optimised@2193da4](https://github.com/aodn/aodn_cloud_optimised/commit/2193da40a0697c861183d422a0b6afe43c42cba8)
> 
> so that the unittest can run ?
> 
> there have been some changes lately, and i got the same bug

Thanks for looking into this, I was wondering what was going on. See additional commit https://github.com/aodn/data-services/pull/1208/commits/79f41e20241d9b8ba60ff1eba24d18da061fe52a

**Review by bpasquer — APPROVED** [review]

**Pull request merged**